cachito/web/api_v1.py

```diff
@@ -27,12 +28,18 @@ def get_requests():
         value exceeds configuration's CACHITO_MAX_PER_PAGE.
     :rtype: flask.Response
     """
+    # Default versbose flag to False
+    verbose = False
```

Comment:
It should be possible to rewrite that as:

```
verbose = str_to_bool(flask.request.args.get('verbose', 'false'))
```

But just an idea, you can keep the current way.